src/index.js

```
@@ -1 +1,17 @@
 'use strict';
+require('dotenv').config({ path: '../.env' });
+
+const { createServer } = require('./createServer');
+const { client } = require('./utils/db');
+
+require('./models/user');
+
+(async () => {
+  try {
+    await client.authenticate();
+
+    await client.sync({ alter: true });
+
+    createServer().listen(5700, () => {});
+  } catch (error) {}
```

The top-level catch block swallows any startup errors (`catch (error) {}`) which will hide problems like DB authentication failures, missing env vars, port conflicts, etc. At minimum log the error and exit with a non-zero status, for example: `console.error(error); process.exit(1);` so CI/ops can detect startup failures.